docs/tests/system_checklist.rst

```diff
@@ -103,11 +103,11 @@ Checklist
       * stage: check Grafana stage dashboard
       * prod: check Grafana prod dashboard
 
-    Is the processor saving crash data to Elasticsearch? To S3?
+    Is the processor saving crash data to Elasticsearch? To cloud storage?
 
     * Check Grafana "processor.es.save_processed_crash"
 
-    * Check Grafana "processor.s3.save_processed_crash"
+    * Check Grafana "processor.storage.save_processed_crash"
```

Comment:
We can't change this unless we're planning to conflate s3 and gcs metrics which I don't think we should do.

We could do something like this:

```
* In AWS:

   * Check Grafana "processor.s3.save_processed.crash"

* In GCP:

   * Check Grafana "processor.gcs.save_processed.crash"
```